src/core_plugins/tile_map/public/maps_visualization.js

```diff
@@ -87,7 +91,15 @@ export function MapsVisualizationProvider(serviceSettings, Notifier, getAppState
       this._kibanaMap.on('zoomchange', () => {
         precisionChange = (previousPrecision !== this._kibanaMap.getAutoPrecision());
         previousPrecision = this._kibanaMap.getAutoPrecision();
-        this.vis.aggs[1].params.precision = previousPrecision;
+        const agg = this._getGeoHashAgg();
```

Comment:
this new guard is breaking these tests: https://github.com/thomasneirynck/kibana/blob/bf63cac183e54f64a054eb781ffba5708270710e/src/ui/public/agg_types/__tests__/buckets/_geo_hash.js#L52-L52

In that mock `vis` object in the test, will need to add mock aggregation.